Makefile

```diff
@@ -165,7 +165,7 @@ coverage: coverage-test ## Run the tests and generate a coverage report.
 coverage-build: all
 	mkdir -p node_modules
 	if [ ! -d node_modules/nyc ]; then \
-		$(NODE) ./deps/npm install nyc --no-save --no-package-lock; fi
+		$(NODE) ./deps/npm install nyc --no-save --no-package-lock --production; fi
```

Comment:
I'm not sure I understand why it's needed, why tests are installed? Running `npm install nyc` does not install tests, we are not running it from a clone of the `nyc` repository.

Reply:
`npm install nyc` will install dev dependencies which aren't needed here. I misspoke (miswrote?) when I wrote "tests". I meant testing libraries and utilities like `tap`. I'll update the commit message.